internal/checker/cloudformation/rules/logging/verify_api_gateway_access_logging_enabled.rego

```diff
@@ -0,0 +1,51 @@
+# METADATA
+# title: "Verify API Gateway Stage has Access Logging enabled"
+# description: "Checks whether API Gateway Stage has Access Logging enabled."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-stage.html
+# custom:
+#   id: CB_CFT_060
+#   severity: LOW
+package lib.cloudformation.CB_CFT_060
+
+import future.keywords.in
+
```

Comment:
update isvalid and add resource
is_valid {
	supported_resources := ["AWS::ApiGateway::Stage", "AWS::Serverless::Api"]
	some resources in input.Resources
	resources.Type in supported_resources
}

resource[type] {
	some resource in input.Resources
    is_valid
    count(fail) > 0
    type := resource.Type
}

resource[type] {
	some resource in input.Resources
    is_valid
    count(pass) > 0
    type := resource.Type
}